lib/game.js

```diff
@@ -1,10 +1,15 @@
+/* eslint-env node */
+
+const uuidV4 = require('uuid/v4');
+
 const { GameEngine } = require('./engine');
 
 
 class Game {
   constructor(server) {
+    this.id = uuidV4();
     this.server = server;
-    this.players = [];
+    this.players = {};
```

Comment:
Here we have a mapping from uuids to class instances of the same type so `Map` should fit.